benchmarking/Dockerfile

```diff
@@ -15,10 +15,11 @@ RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSI
 
 RUN python -c "import clkhash; print('clkhash version:', clkhash.__version__)"
 
+RUN mkdir /cache
 RUN adduser user && \
-    chown user:user /app /var/log
+    chown user:user /app /var/log /cache
+VOLUME /cache
```

Comment:
This is required so that the `user` has permission to write to the mounted folder.